indico/modules/events/tracks/controllers.py

```diff
@@ -17,12 +17,21 @@
 from __future__ import unicode_literals
 
 from indico.modules.events.tracks.forms import TrackForm
+from indico.modules.events.tracks.models.tracks import Track
+from indico.modules.events.tracks.operations import create_track
+from indico.modules.events.tracks.views import WPManageTracks
 from indico.web.util import jsonify_form, jsonify_data
 from indico.util.string import to_unicode
 from MaKaC.webinterface.rh.base import RH
 from MaKaC.webinterface.rh.conferenceModif import RHConferenceModifBase
 
 
+class RHManageTracks(RHConferenceModifBase):
+    def _process(self):
+        tracks = Track.find_all(event_new=self.event_new)
```

Comment:
then it's a good idea to add a position column with the default callback like in other places ;)